backend/node_app/controllers/appStatsController.js

```diff
@@ -544,7 +542,7 @@ class AppStatsController {
 				database: this.constants.MATOMO_DB_CONFIG.database
 			});
 			connection.connect();
-			const results = await this.queryPDFOpenedByUserId('steve', clone_name, connection);
+			const results = await this.queryPDFOpenedByUserId(['steve'], clone_name, connection);
```

Comment:
why is 'steve' hardcoded?

Reply:
not sure removed it to use the userid now